modules/steamos/default.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, ... }:
+
+let
+  inherit (lib)
+    mkOption
+    types
+  ;
+in
+{
+  imports = [
+    ./boot.nix
+    ./mesa.nix
+    ./misc.nix
+    ./sysctl.nix
+  ];
+  options = {
+    jovian.steamos = {
+      useSteamOSConfiguration = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to enable configurations from SteamOS.
```

Comment:
This description is redundant with the name (and equally mysterious).  Can you be more specific about what a "configuration" is?

Reply:
Expanded a bit in the description.